tests/platform_tests/api/test_sfp.py

```diff
@@ -30,7 +31,7 @@
 ]
 
 @pytest.fixture(scope="class")
-def setup(request, duthosts, enum_rand_one_per_hwsku_hostname, xcvr_skip_list, conn_graph_facts):
+def setup(request, duthosts, enum_rand_one_per_hwsku_hostname, xcvr_skip_list, conn_graph_facts, shutdown_ebgp):
```

Comment:
shutdown_ebgp is already present in master [https://github.com/Azure/sonic-mgmt/blob/master/tests/platform_tests/api/test_sfp.py#L34](https://github.com/Azure/sonic-mgmt/blob/master/tests/platform_tests/api/test_sfp.py#L34)